sdk/examples/intkey_go/src/sawtooth_intkey/handler/handler.go

```diff
@@ -4,12 +4,16 @@ import (
 	"crypto/sha512"
 	"encoding/hex"
 	"fmt"
+    "os"
 	cbor "github.com/brianolson/cbor_go"
+    "sawtooth_sdk/logging"
 	"sawtooth_sdk/processor"
 	"sawtooth_sdk/protobuf/processor_pb2"
 	"strings"
 )
 
+var LOGGER *logging.Logger = logging.New(os.Stdout, logging.DEBUG)
```

Comment:
Can we keep the API log4j-like where we set it up to have separation between logger and handlers?  This would then just take a string.